pages/articles/create/index.tsx

```diff
@@ -0,0 +1,264 @@
+import DialogContent from '@mui/material/DialogContent'
+import DialogContentText from '@mui/material/DialogContentText'
+import DialogTitle from '@mui/material/DialogTitle'
+import { useDropzone } from 'react-dropzone'
+const styleduploadbox = {
+  borderWidth: 'thin',
+  borderStyle: 'dashed',
+  borderColor: '#9e9e9e',
+  padding: '20px 0 20px 0',
+}
+const styledtitle = {
+  fontFamily: 'Roboto',
+  fontWeight: 700,
+  margin: '20px 0 20px 0',
+}
+
+const styledtopbar = {
+  display: 'flex',
+  flexDirection: 'row' as const,
+  justifyContent: 'space-between',
+}
+
+const AlertDialog = ({
+  message,
+  open,
+  handleClose,
+}: {
+  message: string
+  open: boolean
+  handleClose: any
+}) => {
+  return (
+    <Dialog open={open} onClose={handleClose}>
+      <DialogTitle id="alert-dialog-title">{'Information'}</DialogTitle>
+      <DialogContent>
+        <DialogContentText id="alert-dialog-description">
+          {message}
+        </DialogContentText>
+      </DialogContent>
+      <DialogActions>
+        <Button onClick={handleClose}>Close</Button>
+      </DialogActions>
+    </Dialog>
+  )
+}
+
+const CreateArticle = () => {
+  const [open, setOpen] = React.useState(false)
+  const [openSave, setOpenSave] = React.useState(false)
+  const [title, setTitle] = useState('')
+  const [file, setFile] = useState<any[]>([])
+  const { getRootProps, getInputProps } = useDropzone({
+    accept: {
+      'image/*': [],
+    },
+    onDrop: (acceptedFiles) => {
+      setFile(
+        acceptedFiles.map((file) =>
+          Object.assign(file, {
+            preview: URL.createObjectURL(file),
+          })
+        )
+      )
+    },
+  })
+  const handleTextChange = (
+    event: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
+  ) => {
+    setTitle(event.target.value)
+  }
+
+  const handleOpen = () => {
+    setOpen(true)
+  }
+  const handleSaveOpen = () => {
+    setOpenSave(true)
+  }
+
+  const handleSaveClose = () => {
+    setOpenSave(false)
+  }
+  const handleClose = () => {
+    setOpen(false)
+  }
+  return (
+    <>
+      <AlertDialog
+        message="Article Published Successfully!"
+        open={open}
+        handleClose={handleClose}
+      />
+      <AlertDialog
+        message="Article Saved Successfully!"
+        open={openSave}
+        handleClose={handleSaveClose}
+      />
+      <div
+        style={{
+          display: 'flex',
+          flexDirection: 'column',
+          height: '100%',
+          padding: '10px',
+        }}
+      >
+        <div style={{ display: 'flex', flexDirection: 'row' }}>
+          <Button onClick={() => router.back()}>
+            <ArrowBackIosNewIcon style={{ margin: 20 }} />
+          </Button>
+          <h4 style={styledtitle}>Create Article</h4>
+        </div>
+        <div>
+          <div style={styledtopbar}>
+            <h6 style={styledtitle}>
+              Title of Article<span style={{ color: 'red' }}>*</span>
+            </h6>
```

Comment:
somehow all the text on this page looks quite different from the other pages. i think you will need to change all the texts to use the `Typography` component instead